### PR TITLE
[PLAT773] List applications for a realm

### PR DESCRIPTION
**Pull request by carrala**
- list all of the applications for a realm 
- or list the applications for a realm with the specified client ids

### CODE REVIEW

**Review by pirtleshell — APPROVED** [review]
looks great! added some style nits 🤪 amazingly thorough application type representation 🏆

File: dist/api/pam.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function listAccessPoliciesForGroups({ realmName, groupIds }, { apiUrl, queenCli
         groupIds.forEach(groupId => query.append('group_ids', encodeURIComponent(groupId)));
         // send request
         const response = yield queenClient.authenticator.tsv1Fetch(`${apiUrl}/v1/identity/pam/policies?${query.toString()}`, { method: 'GET' });
-        const data = yield utils_1.validateRequestAsJSON(response);
+        const data = yield (0, utils_1.validateRequestAsJSON)(response);
```
Comment (rosasck) [inline dist/api/pam.js:23]: @pirtleshell are these 0's config changes for style or why are they showing up now? Just curios what could have caused it

**Review by rosasck — APPROVED** [review]

File: dist/api/pam.js (hunk 1 above)
Comment (pirtleshell) [reply dist/api/pam.js:23]: i have no clue. gonna devote some prof dev time to figure it out this sprint.

**Review by galxy25 — APPROVED** [review]

**Comment by galxy25** [comment]
@carrala can you add a helper method (get/describe/fetch_application_client_by_name that wraps the use of the other new methods for convenience to the developer) please?

**Pull request merged**